### ISSUE TITLE
what about server?

### ISSUE THREAD

**Issue by ghost** [holasebas/instagramclone#1]
what backend does it use? and what are the requirements? please say something about backend / server part

**Comment by holasebas** [comment]
hi, i did not use server backend, i just used static json file info.

**Comment by damilareolowoniyi** [comment]
How would use go about making this application dynamic without using json file. Basically having a backend and a database ?

**Comment by holasebas** [comment]
Yes, you can use an api rest to consume your information and display it in the application. if you dont want to make a back-end server, there is many database service proveiders 
that you can use like Amazon, firebase or mlabs services. good luck!
